next-sitemap.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('next-sitemap').IConfig} */
+
+module.exports = {
+  siteUrl: process.env.SITE_URL || 'https://ethereum.org',
+  generateRobotsTxt: true,
```

Comment:
I see that we are indexing the 404 pages
https://deploy-preview-155--ethereum-org-fork.netlify.app/sitemap-0.xml

Maybe we could exclude those from the sitemap generation by using the `exclude` option.

Reply:
@wackerow I still see some other 404 cases, like the ones for each lang
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/88cc56ce-360a-4a7c-bce3-32755eb1e6a2)

Reply:
Fixed.